# New features
The following are changes cover everything from antiSMASH 6.0 to antiSMASH 7.0.
##Major
<h4>Adenylation domain predictions</h4>
Adenylation domain predictions for NRPS clusters have been updated with a much larger set of known domains from MIBiG 3.1, using signature based methods.
NRPSPredictor2 remains as a SVM/machine learning based predictor.

<h4>CompaRiPPson</h4>
This module compares ribosomally synthesised and post-translationally modified peptide (RiPP) precursors to
known precursors from the MIBiG 3.1 database, along with
predicted precursors from the antiSMASH database.

For more detailed information, see [CompaRiPPson](/modules/comparippson).

<h4>Gene table</h4>
Adds a filterable and interactable table to the HTML output.

Genes can be filtered by entering a search term in the search box (plain text and regex are both supported).
Genes that match the filter will be shown in the region view and, if enabled, the view will automatically zoom to the selection.
Information used for filtering currently includes the name of the gene, its biosynthetic type, and gene function annotations (e.g. SMCoG hits).

Genes can also be interacted with in the table in the same way as in the region view.

<h4>TFBS finder</h4>
A module designed to find transcription factor binding sites, using the [LogoMotif](https://logomotif.bioinformatics.nl) profiles.
The module can be enabled on the command line with `--tfbs`.

For more detailed information, see the [TFBS](/modules/tfbs) page.


##Minor
<h4>Detection rules</h4>
* Added: `2-deoxy-streptamine aminoglycosides`, `aminopolycarboxylic acid metallophores`,
`crocagins`, `methanobactins`, `mycosporines`, `NRP-metallophores`, `opine-like metallophores`,
`phosphonate-likes`, `fungal-RiPP-likes`
* Updated: `phosphonate`

For more detailed information, see the [glossary](/glossary).

<h4>Analysis:</h4>
* New NRPS/PKS domains, including `carnitine-AT (cAT)`, `product template (PT)`, and `thiocysteine/beta-eliminating lyase (SH)`
(see [the full list](/modules/nrps_pks_domains) for details)
* Improved performance and accuracy of unannotated precursor detection in RiPP modules
* Updated ClusterCompare database to MIBiG 3.1
* Clusterblast known database updated to MIBiG 3.1
* Added transATor predictions for Trans-AT PKS domains

<h4>Output:</h4>
* Protoclusters with a cutoff larger than neighbourhood are now marked as being on a contig edge when relevant

<h4>Visualisation:</h4>
* TIGRfam domains within CDS features are now shown in a similar fashion to the existing Pfam and NRPS/PKS domains.
* An additional NRPS/PKS module visualisation focused on domain layout and predicted module ordering
(see [here](/modules/nrps_pks_modules/#bubblemodule-only-drawing))
* Updated KS and Condensation domain NaPDoS links to use NaPDoS v2

<h4>Other:</h4>
* Detection rules can now use dynamic profiles, defined with python code instead of pHMMs
* MUSCLE has been removed as a binary dependency

##Fixes and small changes
<h4>Analysis:</h4>
* Improved detection of cross-CDS NRPS/PKS modules
* Improved non-colinear order finding of pure PKS clusters
* Deconflicted newly found RiPP precursors when multiple types overlapped

<h4>Input handling:</h4>
* Improved handling of splice variants
* Long contig/record names are no longer truncated by default (previous behaviour can be used with `--no-allow-long-headers`)
* Handled some issues when running gene finding over inputs with very large gaps
* Improved handling of various kinds of corrupted/invalid feature annotations in Genbank inputs

<h4>Output:</h4>
* Updated all Pfam links to the new EBI website

<h4>Visualisation:</h4>
* Unified all RiPP precursor sequence visualisations

<h4>Other:</h4>
* Improved various parts of the database download and preparation processes
* Improved file size optimisation for HTML and javascript, for better loading times on slower connections
* Replaced all FTP database download URLs with HTTPS, to better support restricted install environments

Numerous other small changes and fixes were made internally, for a full list see the git shortlog.
