## Important notes

This release increases the minimum python version to 3.9.

# New features
##Minor
<h4>Detection Rules</h4>
* Added rules for darobactin, highly reducing type II PKS, hydrogen cyanide, isocyanide, rSAM-related archaeal RiPPs, triceptide
* Added support for "extender" conditions to rules, for cases where a continguous section of some core genes might be too far from the other core genes to be properly marked
* Added extenders to trans-AT PKS rule to mark all genes in long chains of trans-AT modules as core
* Changed rule categories for larylpolyenes and PUFAs
* Changed ladderane rule to be more specific
* Changed the neighbourhood range of NI-siderophores to be wider in order to cover all of the known clusters

Additionally, new command line options are available to limit detected cluster types to a subset:

* `--hmmdetection-limit-to-rule-names`: restricts protoclusters detected to the named types
* `--hmmdetection-limit-to-rule-categories`: restricts protoclusters detected to the named categories

<h4>Input handling</h4>
* Added support for reusing JSON results compressed with bzip2

<h4>Analysis</h4>
* Updated PFam reference database from 34.0 to 35.0
* Updated GO mappings to 2023-07-05 release
* Updated ClusterBlast reference data from antiSMASH-database version 3.0 to 4.0
* Updated CompaRiPPson reference data from antiSMASH-database version 3.0 to 4.0

<h4>Output</h4>
* Added the product category of protoclusters to their relevant section in JSON results
* Added clusterblast similarity percentage to JSON results
* Removed raw ClusterBlast result file due to size and lack of clarity

<h4>Visualisation</h4>
* Added a link to CDS details that allows BLAST searches against the antiSMASH-database
* Added download link for JSON results file
* Added record versions, where available, to record identifiers in HTML results
* Removed NCBI genomic context links by default, these can be explicitly re-enabled with `--html-ncbi-context`
* Removed links to the now-defunct SEARCHGTr webservice
* Changed lasso peptide and sactipeptide core colouring to only cover cysteines in HTML results
* Changed CompaRiPPson characters in consensus to use the same colouring as in the queries, where relevant

##Fixes and small changes
<h4>Input handling</h4>
* Changed all generated CDS translations to start with methionine, even when using an alternative start codon

<h4>Analysis</h4>
* Removed some rare instances of fused lanthipeptide genes being interpreted as a precursors
* Changed lasso peptide precursor handling to avoid motif duplication when shared by two protoclusters
* Changed trans-AT PKS module classification to be less strict
* Changed lanthipeptide tailoring enzyme detection to be less strict
* Changed all RiPP precursor ORF finding to be less strict
* Changed capreomycidine substrate SMILES to reorder the backbone

<h4>Output</h4>
* Added error output to log files, when relevant

<h4>Visualisation</h4>
* Changed default ClusterCompare comparison mode shown from ProtoclusterToRegion to RegionToRegion

Numerous other small changes and fixes were made internally, for a full list see the git shortlog.
